sympy/series/gruntz.py

```diff
@@ -553,6 +553,18 @@ def mrv_leadterm(e, x):
     #
     w = Dummy("w", positive=True)
     f, logw = rewrite(exps, Omega, x, w)
+    from sympy.core.power import Pow
+    li =[]
```

Comment:
Let's try to follow spacing uniformly in the codebase.
```suggestion
    li = []
```